test/autorest/generated/lrogroup/pollers.go

```diff
@@ -4551,200 +88,84 @@ func (p *lrosaDsPut200InvalidJSONPoller) ResumeToken() (string, error) {
 	return string(js), nil
 }
 
-// Wait will continue to poll until a terminal state is reached or an error is encountered. Wait will use the
-// duration specified in the retry-after header, if the header is not specified then the pollingInterval that
-// is specified will be used to wait between polling requests.
-func (p *lrosaDsPut200InvalidJSONPoller) Wait(ctx context.Context, pollingInterval time.Duration) (*ProductResponse, error) {
-	for p.Poll(ctx) {
-		if delay := p.response().RetryAfter(); delay > 0 {
-			time.Sleep(delay)
-		} else {
-			time.Sleep(pollingInterval)
-		}
-	}
-	return p.Response()
+func httpPollerHandleResponse(resp *azcore.Response, p azcore.Pipeline) (*HttpResponse, error) {
+	result := &HttpResponse{RawResponse: resp.Response}
+	return result, nil
 }
 
-// response returns the last HTTP response.
-func (p *lrosaDsPut200InvalidJSONPoller) response() *azcore.Response {
-	return p.pt.latestResponse()
+// httpPoller handles the error response.
+func httpPollerHandleError(resp *azcore.Response) error {
```

Comment:
This doesn't appear to be referenced.

Reply:
You're right, I updated it. It's not being referenced when creating the polling trackers.

Reply:
Can't we use the one on the client and omit this one?

Reply:
Yes we can, I cant remember why this was separated.

Reply:
Done